app/components/shared/Header/Header.tsx

```diff
@@ -68,11 +69,13 @@ export const Header: React.FC<HeaderProps> = ({ user, nav = "left", children })
       {
         id: "login",
         el: () => (
-          <Form action="/api/auth/auth0" method="post">
-            <Button type="submit" variant="outline">
-              Login
-            </Button>
-          </Form>
+          <Button
+            type="submit"
+            variant="outline"
+            onClick={() => fetcher.load("/api/auth/auth0")}
```

Comment:
This fetcher.load is a GET call. Looking at the code in "api/auth/auth0" the logic to login is slightly different between the GET and POST methods. We should probably update that api so that the GET method also handles the   prompt logic.

url.searchParams.append("prompt", "login")

Reply:
Sure, I agree with you; the problem that we were having with the app crashing was related to the redirects returned by `/api/auth/auth0`. I like the idea that you're presenting, will work on it.

Reply:
im not seeing how this was resolved...?

Reply:
Updated